docs/basics/connection-approval.md

```diff
@@ -25,6 +25,7 @@ This is how the connection approval response is formed by server-side specific u
 - **PlayerPrefabHash**: The type of player prefab to use for the authorized player (_if this is null it uses the default `NetworkManager` defined player prefab_)
 - **Position** and **Rotation**: The position and rotation of the player when spawned
 - **Pending**: Provides the ability to mark the approval "pending" to delay the authorization until other user-specific code finishes the approval process.
+- **Reason**: If `Approved` is `false` you can populate this with a `string` based message (or `json`) to send the reason why the client was not approved.
 
```

Comment:
- **PlayerPrefabHash**: The type of player prefab to use for the authorized player. It uses the default `NetworkManager` defined player prefab if it's null.
- **Position** and **Rotation**: The position and rotation of the player when spawned.
- **Pending**: Provides the ability to mark the approval as "pending" to delay the authorization until other user-specific code finishes the approval process.
- **Reason**: If `Approved` is `false`, you can populate the `Reason` with a `string`-based message (or JSON) explaining why the client wasn't approved.